pycarol/connectors.py

```diff
@@ -362,3 +365,272 @@ def get_dm_mappings(self, connector_id=None, connector_name=None, staging_name=N
+                When using with consumer.False if you don't know what consumer is.
+            staging_name: str` default None
+                Name of the staging to find the mapping. Returns 404  if there is not mapping..
+            offset: `int`, default 0
+                Offset for pagination. Only used when `scrollable=False`
+            page_size: `int`, default 100
+                Number of records downloaded in each pagination. The maximum value is 1000
+            sort_order: `str`, default 'ASC'
+                Sort ascending ('ASC') vs. descending ('DESC').
+            sort_by: `str`,  default `None`
+                Name to sort by.
+            print_status: `bool`, default `False`
+                Print the number of records in each interaction.
+            errors: {‘ignore’, ‘raise’}, default ‘raise’
+                If ‘raise’, then invalid request will raise an exception If ‘ignore’,
+                then invalid request will return the request response.
+
+        Returns: list of dict
+            List of dict with mappings,
+        """
+
+        connector_id = connector_id if connector_id else self.get_by_name(connector_name)['mdmId']
+
+        template_data = []
+        count = offset
+        query_params = {
+            "offset": offset, "pageSize": str(page_size),
+            "sortOrder": sort_order,
+            "sortBy": sort_by,
+            'stagingType': staging_name,
+            'reverseMapping': reverse_mapping,
+        }
+
+        set_param = True
+        total_hits = float("inf")
+        while count < total_hits:
+            query = self.carol.call_api(path=f'v1/connectors/{connector_id}/entityMappings', method="GET",
+                                        params=query_params, errors=errors)
+
+            if query.get('hits') is None:
+                if len(template_data) == 0:
+                    # when errors==ignore it will return the error msg.
+                    return query
+                else:
+                    return template_data
+
+            if query['count'] == 0:
+                print('There are no more results.')
+                print(f'Expecting {total_hits}, response = {count}')
+                break
+            count += query['count']
+            if set_param:
+                total_hits = query["totalHits"]
+                set_param = False
+
+            query = query['hits']
+            template_data.extend(query)
+
+            query_params['offset'] = count
+            if print_status:
+                print(f'{count}/{total_hits}', end='\r')
+
+        return template_data
+
+    def _play_pause_mapping(self, kind, entity_mapping_id=None, staging_name=None,
+                            connector_name=None, connector_id=None,
+                            reverse_mapping=False,
+                            process_cds=True, ):
+
+        connector_id = connector_id if connector_id else self.get_by_name(connector_name)['mdmId']
```

Comment:
on Delete operation there is a check and a deprecation message.
I think both should have the same behaviour right?